### DISCUSSION TITLE
UI interface suggestions

### DISCUSSION THREAD

**Discussion by weasonn** [rustdesk/rustdesk#5118] [Feature Request]
Hi there,

I have 3 suggestions below.

1. Rustdesk has 2 separated windows. It should be combined together like the Finalshell. Check the pic below.

2. The tabs of Rustdesk cannot be dragged or detached or attached. It makes people feel very troublesome.

3. When we click the close button, Rustdesk closes itself directly. But I hope it stay in the taskbar notification area. 

![0725193853](https://github.com/rustdesk/rustdesk/assets/27633819/c18e8ecb-811f-4906-b0fe-6dadf2409861)

**Comment by dinger1986** [comment]
Hello,

1. Most systems on the market are exactly the same and launch another window. I for one prefer it.
2. I believe detaching tabs has already been discussed. 
3. I guess a tick box to close to tray would be good but as it's a service it's always running anyway so doesn't make a huge difference.

**Reply by weasonn** [reply]
Hello,
1. I think we should make things simpler. That's the meaning of the Tabs. (I don't prefer the taskbar has 2 same tasks, because i may choose the wrong one if i don't distinguish carefully)
2. I checked the discussion #2889. But he said Seperate window mode will be added. I think it's not the tabs reordering or detaching.
3. Close to tray is really a good idea for most people. At least you can give us an option to choose between close app or close to tray.
Thanks
